components/breadcrumb/src/Breadcrumb.js

```diff
@@ -13,23 +13,23 @@ import { SrOnly } from '@westpac/accessibility-helpers';
  * Breadcrumb: Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.
  */
 export const Breadcrumb = ({ children, ...props }) => {
-	const { breadcrumb } = useTheme();
-
-	const common = {
-		padding: breadcrumb.padding,
-		marginBottom: breadcrumb.marginBottom,
-		fontSize: breadcrumb.fontSize,
-		listStyle: 'none',
-	};
-
 	const childrenWithProps = Children.map(children, (child, index) =>
-		index === Children.count(children) - 1 ? cloneElement(child, { last: 'true' }) : child
+		index === Children.count(children) - 1 ? cloneElement(child, { last: true }) : child
```

Comment:
TL;DR: This is a valid use case for `cloneElement`, despite me recommending not using it.

I keep suggesting to not using `cloneElement`, but when dealing with things you'd do with `Children.count`, as for example here to determine the last child, I think this `cloneElement` is actually the way to go.

If you find yourself in need to know the children index, this is a good use for it.

If you're just passing props that you could grab from the child via Context, I recommend using Context instead (more robust to DOM structure change, doesn't break React's composition model).

Reply:
Very well explained as always @simonswiss

How about @florettewest’s question. When using cloneElement() should we check the type of the child to ensure props aren’t being passed to random children?

Reply:
I would say this is a great safe guard. Make sure the correct child is passed, and trow an error when it's not, to prevent accidents from happening.